src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
@powercode, your last commit had 8 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Basic SysLog tests on Linux.Verifies logging level filtering works
```powershell
Expected $null, but got PSLogItem.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 256
256:         $items | Should -Be $null
```
SemanticVersion api tests.Error handling.Missing parts: 'NullString': ''
```powershell
Expected an exception, with FullyQualifiedErrorId 'PSArgumentNullException' to be thrown, but the FullyQualifiedErrorId was 'FormatException'. from /home/vsts/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1:206 char:15
    +             { [SemanticVersion]::new($version) } | Should -Throw -Err ?
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1: line 206
206:             { [SemanticVersion]::new($version) } | Should -Throw -ErrorId $errorId
```
Alias tests.Export-Alias literal path.for CSV
```powershell
Expected <empty>, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/alias.tests.ps1: line 57
57:                     $exportAliasError.FullyqualifiedErrorId | Should -Be $test.expectedError
```
Alias tests.Export-Alias literal path.for PS1
```powershell
Expected <empty>, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/alias.tests.ps1: line 57
57:                     $exportAliasError.FullyqualifiedErrorId | Should -Be $test.expectedError
```
Alias tests.Export-Alias literal path.for Null
```powershell
Expected strings to be the same, but they were different.
Expected length: 95
Actual length:   102
Strings differ at index 32.
Expected: '...ErrorNullN...'
But was:  '...ErrorEmpty...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/alias.tests.ps1: line 57
57:                     $exportAliasError.FullyqualifiedErrorId | Should -Be $test.expectedError
```